zipline/data/data_portal.py

```diff
@@ -59,6 +59,362 @@
 HISTORY_FREQUENCIES = set(["1m", "1d"])
 
 
+class DailyHistoryAggregator(object):
```

Comment:
I see your point. On the other hand the `DailyHistoryAggregator` is a support class for the `DataPortal`.

As we discussed in person, that could mean that the name of the module itself could renamed to not imply that it is just data_portal.

Either way, going to defer/punt on that refactoring.